src/OrbitGgp/MockGgp/Working.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <chrono>
+#include <cstring>
+#include <iostream>
+#include <thread>
+
+int GgpVersion(char* argv[]) {
+  if (strcmp(argv[1], "version") != 0) {
```

Comment:
I think we should refrain from using C-functions and stick to C++. In this case that would mean casting to `std::string_view` and use its internal comparison functionality. Something like: `std::string_view{argv[1]} == "version"`.

Reply:
done